feature-libs/asm/assets/translations/en/asm.json

```diff
@@ -6,7 +6,7 @@
     "hideUi": "Close ASM",
     "helpPortalUrl": "Help",
     "customers": "Customers",
-    "asmCustomer360Button": "Customer 360 View",
+    "asmCustomer360Button": "Customer 360°",
```

Comment:
create new instead of chaning existing one

Reply:
have been modified.